tests/phpunit/includes/TestCase.php

```diff
@@ -80,6 +80,11 @@ public function tear_down() {
 		parent::tear_down();
 		// Clear screen related globals.
 		unset( $GLOBALS['current_screen'], $GLOBALS['taxnow'], $GLOBALS['typenow'] );
+
+		// Clean up scripts hooks to avoid interefence between tests
```

Comment:
As above, plus there's a typo:

```suggestion
		// Clean up scripts hooks to avoid interference between tests.
```